consultation_analyser/consultations/migrations/0001_initial.py

```diff
@@ -1,19 +1,85 @@
-# Generated by Django 5.0.2 on 2024-02-29 14:08
+# Generated by Django 5.0.3 on 2024-03-05 16:48
```

Comment:
Having tried to apply this locally I needed to clear my db first — I think fine to have multiple migrations!